docs/basics.rst

```diff
@@ -321,7 +321,7 @@ The user can provide these additional parameters:
 
     - ``--dev`` — Install both ``develop`` and ``default`` packages from ``Pipfile``.
     - ``--system`` — Use the system ``pip`` command rather than the one from your virtualenv.
-    - ``--deploy`` — The workflow is using pipenv for package loc in local development environment for having a Pipfile and Pipfile.lock. The ``--deploy`` makes sure the packages are properly locked in Pipfile.lock.
+    - ``--deploy`` — The workflow is using pipenv for package loc in local development environment for having a Pipfile and Pipfile.lock. The ``--deploy`` makes sure the packages are properly locked in Pipfile.lock, and abort if the lock file is out-of-date.
```

Comment:
The first sentence is meaningless, To be consistent, just
`— Make sure the packages are properly locked in Pipfile.lock, and abort if the lock file is out-of-date.` is enough